lib/Entity/DataSet.php

```diff
@@ -492,6 +492,52 @@ public function getData($filterBy = [], $options = [])
             'connection' => 'default'
         ], $options);
 
+        // Fetch display tag value/s
+        if ($filter != '' && $displayId != 0) {
+            $displayTags = [];
+
+            // Define the regular expression to match [Tag:tagNameHere]
+            $pattern = '/\[Tag:(\w+)\]/';
+
+            // find all matches
+            preg_match_all($pattern, $filter, $matches);
+
+            // Check if matches were found
+            if (!empty($matches[1])) {
+                // Iterate through the matches and add them to the displayTags array
+                foreach ($matches[1] as $tag) {
+                    $displayTags[] = $tag;
+                }
+            }
+
+            // Loop through each tag and get the actual tag value from the database
+            foreach ($displayTags as $tag) {
+                $query = "
```

Comment:
String should be enclosed in `'` instead of `"` please

Reply:
Done!